### ISSUE TITLE
[BUG] Get-PnPSiteTemplate: The given key was not present in the dictionary.

### ISSUE THREAD

**Issue by KellyDJones** [pnp/powershell#109]
### Reporting an Issue or Missing Feature
When I use Get-PnPSiteTemplate, I'm receiving "the given key was not present in the dictionary".  This is the same error that I was getting with Invoke-PnPSiteTemplate until it was fixed in last night's build (0.3.33).

### Actual behavior 
An error displayed: "Get-PnPSiteTemplate: The given key was not present in the dictionary."

### Steps to reproduce the behavior
```
Connect-PnPOnline -Url {siteUrl} -ClientId {clientID} -Tenant {tenant} -CertificatePath ..\{fileName}.pfx
Get-PnPSiteTemplate -Out {filename}.xml -PersistBrandingFiles -PersistPublishingFiles -IncludeAllPages
```

### What is the version of the Cmdlet module you are running?
```
ModuleType Version    PreRelease Name                                PSEdition
---------- -------    ---------- ----                                --------- 
Manifest   0.3.33     nightly    PnP.PowerShell                      Core,Desk
```

### Which operating system/environment are you running PnP PowerShell on?
- [ X] Windows

**Comment by erwinvanhunen** [comment]
Thank you! We indeed missed updating a part of the code. It has been fixed now and will be available in release 0.3.34-nightly.

**Comment by KellyDJones** [comment]
Thank you!
